### PR TITLE
Fixing cruft that was added with React

### PR DESCRIPTION
**Pull request by marcomagdy**
This allows the backend to be used with curl

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -2,25 +2,15 @@
...
-builder.Services.AddControllersWithViews();
+builder.Services.AddControllers();
 builder.Services.AddSingleton<DapperContext>();
 var app = builder.Build();
 
-// Configure the HTTP request pipeline.
-if (!app.Environment.IsDevelopment())
-{
-    // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
-    app.UseHsts();
```
Comment (marcomagdy) [inline Program.cs:13]: I had no idea what this was, and I looked it up, and it's one of those "extra security" that we don't care about for this project.

File: Program.cs [hunk 2]
```diff
@@ -2,25 +2,15 @@
 
 // Add services to the container.
 
-builder.Services.AddControllersWithViews();
+builder.Services.AddControllers();
```
Comment (marcomagdy) [inline Program.cs:5]: This is for MVC projects. This is not such a project.

File: Properties/launchSettings.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
     "psaltos": {
       "commandName": "Project",
       "launchBrowser": true,
-      "applicationUrl": "https://localhost:7226;http://localhost:5279",
+      "applicationUrl": "http://localhost:5279",
```
Comment (marcomagdy) [inline Properties/launchSettings.json:14]: This is what I believe was ruining the curling behavior

**Review by AthanIsaac — APPROVED** [review]
LGTM

**Pull request merged**